.github/workflows/lint.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         python-version: 3.8
 
     - name: Install dependencies
-      run: pip install -U flake8 wheel setuptools yapf==0.32.0 toml
+      run: pip install --upgrade flake8 wheel setuptools yapf==0.32.0 toml
```

Comment:
Given https://docs.python.org/3.11/library/tomllib.html why not use https://pypi.org/project/tomli instead of https://pypi.org/project/toml on Python < 3.11 and tomllib on Python >= 3.11?
* https://peps.python.org/pep-0680

Reply:
The `toml` dependency is due to `yapf`, and I don't think that supports `tomllib` (yet?). 

With next `yapf` release, we'll be able to install [`yapf[pyproject]`](https://github.com/google/yapf/blob/212c5b5ad8e172d2d914ae454c121c89cccbcb35/setup.py#L84) instead...